src/core/remote.js

```diff
@@ -2065,8 +2065,7 @@ Remote.prototype.requestRippleBalance = function(options, callback) {
 
     // accountHigh implies for account: balance is negated.  highLimit is the
     // limit set by account.
-    const accountHigh = UInt160.from_json(options.account)
-    .equals(highLimit.issuer());
+    const accountHigh = options.account.equals(highLimit.issuer());
```

Comment:
Good catch, should be string addresses: `options.account === highLimit.issuer()`